addons/project/data/project_demo.xml

```diff
@@ -252,6 +284,28 @@
             <field name="project_id" ref="project.project_project_1"/>
         </record>
 
+        <!-- Project Home Construction -->
+        <record id="project_home_construction_milestone_1" model="project.milestone">
+            <field name="is_reached" eval="True"/>
+            <field name="deadline" eval="time.strftime('%Y-%m-2')"/>
+            <field name="name">Piping Installation</field>
+            <field name="reached_date" eval="time.strftime('%Y-%m-2')"/>
+            <field name="project_id" ref="project.project_home_construction"/>
+        </record>
+        <record id="project_home_construction_milestone_2" model="project.milestone">
+            <field name="is_reached" eval="False"/>
+            <field name="deadline" eval="time.strftime('%Y-%m-5')"/>
+            <field name="name">Door and Window Framing</field>
+            <field name="project_id" ref="project.project_home_construction"/>
+        </record>
+        <record id="project_home_construction_milestone_3" model="project.milestone">
+            <field name="is_reached" eval="False"/>
+            <field name="deadline" eval="time.strftime('%Y-%m-10')"/>
+            <field name="name">Wiring</field>
+            <field name="reached_date" eval="time.strftime('%Y-%m-10')"/>
```

Comment:
As `is_reached` is False, does `reached_date` make sense?

Reply:
Right. :+1: